src/views/cg-view.test.tsx

```diff
@@ -0,0 +1,273 @@
+
+		it('should hide send mail hover action when the contact group has 0 members', async () => {
+			const openMailComposer = jest.fn();
+			jest.spyOn(shell, 'useIntegratedFunction').mockReturnValue([openMailComposer, true]);
+			const contactGroupName = faker.company.name();
+			registerFindContactGroupsHandler({
+				findContactGroupsResponse: createFindContactGroupsResponse([
+					createFindContactGroupsResponseCnItem(contactGroupName, [])
+				]),
+				offset: 0
+			});
+			setupTest(<CGView />);
+
+			await screen.findAllByText(contactGroupName);
+			expect(screen.queryByTestId(TESTID_SELECTORS.icons.sendEmail)).not.toBeInTheDocument();
+		});
+
+		it('should open the mail board (Contextual menu trigger)', async () => {
+			const openMailComposer = jest.fn();
+			jest.spyOn(shell, 'useIntegratedFunction').mockReturnValue([openMailComposer, true]);
+			const contactGroupName = faker.company.name();
+			const member = faker.internet.email();
+			registerFindContactGroupsHandler({
+				findContactGroupsResponse: createFindContactGroupsResponse([
+					createFindContactGroupsResponseCnItem(contactGroupName, [member])
+				]),
+				offset: 0
+			});
+			const { user } = setupTest(<CGView />);
+
+			await screen.findByText(contactGroupName);
+			const listItem = await screen.findByTestId('list-item-content');
+			await user.rightClick(listItem);
+
+			const contextualMenu = await screen.findByTestId(TESTID_SELECTORS.dropdownList);
+			const action = within(contextualMenu).getByText('Send e-mail');
+			await user.click(action);
+			expect(openMailComposer).toHaveBeenCalledTimes(1);
+			expect(openMailComposer).toHaveBeenCalledWith({ recipients: [{ email: member }] });
+		});
+	});
+
+	describe('Displayer', () => {
+		it('should show and the empty displayer message as default', async () => {
+			registerFindContactGroupsHandler({
+				findContactGroupsResponse: createFindContactGroupsResponse([]),
+				offset: 0
+			});
+			setupTest(<CGView />);
+
+			expect(await screen.findByText(EMPTY_DISPLAYER_HINT)).toBeVisible();
+		});
+
+		it('Click on a list item open the displayer for that item', async () => {
+			const contactGroupName = faker.company.name();
+			registerFindContactGroupsHandler({
+				findContactGroupsResponse: createFindContactGroupsResponse([
+					createFindContactGroupsResponseCnItem(contactGroupName)
+				]),
+				offset: 0
+			});
+			const { user } = setupTest(
+				<Route path={`${ROUTES.mainRoute}${ROUTES.contactGroups}`}>
+					<CGView />
+				</Route>,
+				{ initialEntries: [`/${ROUTES_INTERNAL_PARAMS.route.contactGroups}`] }
+			);
+			await screen.findByText(contactGroupName);
+			await screen.findByText(EMPTY_DISPLAYER_HINT);
+			await user.click(screen.getByText(contactGroupName));
+			await screen.findByRoleWithIcon('button', { icon: TESTID_SELECTORS.icons.closeDisplayer });
+			expect(screen.getAllByText(contactGroupName)).toHaveLength(3);
+			// FIXME
+			// expect(screen.getByText(/addresses list/i)).toBeVisible();
```

Comment:
Can this FIXME be resolved? If no, is the expect needed? If yes, please add a comment to explain why there is the fixme. Otherwise, remove these two lines.